packages/sandbox/src/index.ts

```diff
@@ -314,12 +314,15 @@ export const createTypeScriptSandbox = (
     return (firstJS && firstJS.text) || ""
   }
 
+  const isDtsFile = (name: string) => /\.d\.[cm]?ts$/.test(name)
```

Comment:
You may also want to support `.d.*.ts`, e.g. `.d.css.ts`. https://github.com/microsoft/DefinitelyTyped-tools/blob/7a8a32513cbbe5a2af5c3e39e056117b3c14c390/packages/utils/src/miscellany.ts#L97

Reply:
Interesting, good to know. Updated